### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by lizardlynx**

### CODE REVIEW

File: Database/funcs/UpdateReview.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+-- =============================================
+CREATE PROCEDURE [dbo].[UpdateReview]
+	-- Add the parameters for the stored procedure here
+	-- Add the parameters for the stored procedure here
+@ReviewID int,
+@UserID  int,
+@Hotel int,
+@Rating int,
+@Review nvarchar(MAX),
+@Date int
```
Comment (VsIG-official) [inline Database/funcs/UpdateReview.sql:29]: Is date should be "int"? 'Cause in [https://camo.githubusercontent.com/e780e0495d065160464be133b87375fd6ec724e2b30a2c2a01feb61cc97db936/68747470733a2f2f692e6962622e636f2f384b6e334a66422f426f6f6b696e672e706e67](url) We use "date"

**Review by VsIG-official — COMMENTED** [review]
date or int?

**Comment by mezgoodle** [comment]
Can you move your **.sql** files from _funcs_ directory to _Procedures_ dir?

File: Database/funcs/UpdateReview.sql (hunk 1 above)
Comment (lizardlynx) [reply Database/funcs/UpdateReview.sql:29]: We have decided that @Date should be of type date

**Comment by mezgoodle** [comment]
Is it bad pr?

**Pull request closed without merging**